src/Microsoft.Experimental.Collections/Microsoft/Collections/Extensions/RefDictionary.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Lincense headers should be:

```cs
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

Reply:
I know you got the headers from MultiValueDictionary implementation. I will submit a PR to fix those.